docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,75 @@
+# Set up a database for Scalar DB/Scalar DL deployment in AKS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment in AKS.
+
+## Cosmos DB
+
+In this section, you will create a Cosmos DB account.
+
+### Requirements
+
+* You must create a Cosmos DB account with `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Recommendations
+
+* You should configure the backup policy as `Continuous` for Point-in-time-restore (PITR).
+* You should configure the service endpoint based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint) to restrict access to Cosmos DB for production.
+
+### Steps
+
+* Create an Azure Cosmos DB account based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
```

Comment:
Azure official guide doesn't explain anything for Scalar specific setting but is it OK for users?
Not need to set anything for Scalar?
Configuration is done in ConfigureScalarDB.md?

Reply:
Please answer my questions properly.
I don't get the answers from your comments.

Reply:
> Azure official guide doesn't explain anything for Scalar specific setting but is it OK for users?

Azure official guide doesn't explain anything Scalar specific setting. We have mentioned the essential settings in Requirements section, which a user can select when following the Azure official guide.

> Not need to set anything for Scalar?

We have added the essential settings required for scalar in the Requirements section. So we are not repeating the same in the steps. We have mentioned it in `Performance Scaling` section regarding `cosmosBaseResourceUnit`. I will add it as part of the `Steps` section when I update the structure

> Configuration is done in ConfigureScalarDB.md?

The configuration required to use the Cosmos DB is explained in ConfigureScalarDB.md and ConfigureScalarDL.md files.